types/fastify-multipart/index.d.ts

```diff
@@ -0,0 +1,66 @@
+// Type definitions for fastify-multipart 0.5
+// Project: https://github.com/fastify/fastify-multipart#readme
+// Definitions by: Jannik Keye <https://github.com/jannikkeye>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+/// <reference types="node" />
+/// <reference types="busboy" />
```

Comment:
I think triple-slash references are only used for node in typings.  Maybe this should be `import busboy = require("busboy");`?

Reply:
Indeed, wonder why I did this as I did it correctly in a another PR already 😬